src/authService.js

```diff
@@ -63,28 +83,97 @@ export class AuthService {
 
   login(email, password) {
     let loginUrl = this.auth.getLoginUrl();
-    let content;
+    let config = this.config;
+    let clientId = this.config.clientId;
+    let content = {};
+    let options = {};
+    let data = [];
     if (typeof arguments[1] !== 'string') {
       content = arguments[0];
     } else {
-      content = {
+      content = clientId ?  {
+        'email': email,
+        'password': password,
+        'client_id': clientId
+      }
+      : {
         'email': email,
         'password': password
       };
     }
 
-    return this.client.post(loginUrl, content)
+    if (this.config.postContentType === 'json') {
```

Comment:
Not sure about this code. It might have use, but to me the problem is in aurelia-api:

https://github.com/SpoonX/aurelia-api/blob/master/src/rest.js#L39

Which will always json stringify if it's an object (so I don't see the point in doing it here). What I do like is the support for a different content-type. But perhaps that should be moved to aurelia-api? @doktordirk your 2 cents?